apps/api/src/app/validate/validate.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Body, ClassSerializerInterceptor, Controller, UseInterceptors, Post } from '@nestjs/common';
+import { ExternalApiAccessible } from '../auth/framework/external-api.decorator';
+import { ApiCommonResponses, ApiResponse } from '../shared/framework/response.decorator';
+import { UserAuthentication } from '../shared/framework/swagger/api.key.security';
+import { ValidateBridgeUrlRequestDto } from './dtos/validate-bridge-url-request.dto';
+import { ValidateBridgeUrlResponseDto } from './dtos/validate-bridge-url-response.dto';
+import { ValidateBridgeUrlCommand } from './usecases/validate-bridge-url/validate-bridge-url.command';
+import { ValidateBridgeUrlUsecase } from './usecases/validate-bridge-url/validate-bridge-url.usecase';
+
+@ApiCommonResponses()
+@Controller('/validate')
```

Comment:
Should we maybe just have this on the existing bridge controller instead of a new resourcE?

Reply:
makes sense, didn't think to look at the ee controllers, will move it

Reply:
![CleanShot 2024-06-24 at 09 25 23@2x](https://github.com/novuhq/novu/assets/8872447/cc74655b-6f00-4c8f-ac81-8e19c445ca86)
There is an existing `status` endpoint 😅

Reply:
ye, but this one comes from the environment, the case I have supplies the endpoint. I will see if there is a a way to consolidate if needed